src/clappr-dash-shaka-playback.js

```diff
@@ -123,6 +123,88 @@ class DashShakaPlayback extends HTML5Video {
     this._onAdaptation()
   }
 
+  /**
+   * @override
+   */
+  get closedCaptionsTracks() {
+    let id = 0
+    let trackId = () => { return id++ }
+    let tracks = this.textTracks || []
+
+    return tracks
+      .filter(track => track.kind === 'subtitle')
+      .map(track => { return {id: trackId(), name: track.label || track.language, track: track} })
+  }
+
+  /**
+   * @override
+   */
+  get closedCaptionsTrackId() {
+    return super.closedCaptionsTrackId
+  }
+
+  /**
+   * @override
+   */
+  set closedCaptionsTrackId(trackId) {
+    if (!this._player) {
+      return
+    }
+
+    let tracks = this.closedCaptionsTracks
+    let showingTrack
+
+    // Note: -1 is for hide all tracks
+    if (trackId !== -1) {
+      showingTrack = tracks.find(track => track.id === trackId)
+      if (!showingTrack) {
+        return // Track id not found
+      }
+      if (this._shakaTTVisible && showingTrack.track.active === true) {
+        return // Track already showing
```

Comment:
Dido, but for sure, not showing up all of them on the console, maybe using INFO severity.